CONTRIBUTING.md

```diff
@@ -1,71 +1,21 @@
 [code_of_conduct]: ./CODE_OF_CONDUCT.md
-[repo_from_template]: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template
 [fork]: https://docs.github.com/en/github/getting-started-with-github/fork-a-repo
 [clone]: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository
 [push]: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/merging-an-upstream-repository-into-your-fork
 [pull]: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork
-[template]: https://github.com/ERNI-Academy/erni-academy-template
 
 # Contributing
 
 This document is a set of guidelines for contributing to this repository.
 
-- [Contributing](#contributing)
-  - [Getting started](#getting-started)
-  - [1. How to become a contributor](#1-how-to-become-a-contributor)
-  - [2. How to create a resource](#2-how-to-create-a-resource)
-  - [3. How to contribute](#3-how-to-contribute)
-  - [Code of Conduct](#code-of-conduct)
-
-## Getting started
-
-Please note that this project is released with a Contributor [Code of Conduct][code_of_conduct]. By participating in this project you agree to abide by its terms.
-
-All contributions, however small are valued!
-
-If you have an idea to share in the form of Proof of concept, Starterkit or Guideline become a contributor and collaborate with us.
-
-## 1. How to become a contributor
-
-Before starting you must be a ERNI contributor.
-
-Please send us an email to this account so we can add you to our contributors list.
-
-📧 <esp-services@betterask.erni>
-
-## 2. How to create a resource
```

Comment:
We are losing this section, right now is our "naming convention".  Are we good with this?